source/change-streams/change-streams.rst

```diff
@@ -167,7 +207,13 @@ Driver API
     /**
      * The resume token (_id) of the document the iterator last returned
      */
-    private resumeToken: Document;
+    private documentResumeToken: Document;
+
+    /**
+     * The most recent postBatchResumeToken returned in an aggregate or
+     * getMore response. For pre-4.2 versions of MongoDB, this remains unset.
+     */
+    private postBatchResumeToken: Document;
```

Comment:
I'm contradicting the decision made in [these comments](https://github.com/mongodb/specifications/pull/413#discussion_r231523141) to better distinguish the resume token sourced from the document _id and the resume token sourced from postBatchResumeToken.